python/ingestion/dataset_utils.py

```diff
@@ -202,7 +201,7 @@ def add_sum_of_rows(df, breakdown_col, value_col, new_row_breakdown_val,
        df: The DataFrame to calculate new rows from.
        breakdown_col: The name of the breakdown column that a new value is being
                       summed over.
-       value_col: The name of the column whose values should be summed.
+       value_col: The name or names of the column/s whose values should be summed.
```

Comment:
can we specify the expected types here in the doc string since we're overloading this arg? just like, `the string name (or list of string names) of the column/s...` and i dont know enough about python typing but can you indicate str OR str[] in the function signature?